### PR TITLE
Character jason marxsen

### PR DESCRIPTION
**Pull request by TempusEstNunc**
Added a dictionary containing character ID and character objects corresponding to each character in the story

### CODE REVIEW

File: Project/MyGameLibrary/Character/CharacterCollection.cs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+namespace MyGameLibrary.Character
+{
+    public class CharacterCollection
+    {
+        public Dictionary<characterID, Character> CharacterDictionary;
+
+        private Dictionary<Items, int> dict1;
+        private Dictionary<Items, int> dict2;
+        private Dictionary<Items, int> dict3;
```
Comment (HayleyO) [inline Project/MyGameLibrary/Character/CharacterCollection.cs:17]: these need to be named better

File: Project/Fall2020_CSC403_Project/TextEngine.cs [hunk 2]
```diff
@@ -6,12 +6,16 @@
...
+using MyGameLibrary.Shop;
+using MyGameLibrary.Story;
+using MyGameLibrary.Character;
 
 namespace Fall2020_CSC403_Project
 {
     public partial class TextEngine : Form
     {
         private Story Story { get; set; }
+        private CharacterCollection Characters = new CharacterCollection();
```
Comment (HayleyO) [inline Project/Fall2020_CSC403_Project/TextEngine.cs:18]: Don't create the character collection object

File: Project/MyGameLibrary/Character/CharacterCollection.cs [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        private Dictionary<Items, int> dict4;
+        private Dictionary<Items, int> dict5;
+
+        private Character Wendy;
+        private Character Tony;
+        private Character Ronald;
+        private Character Green;
+        private Character BurgerKing;
```
Comment (HayleyO) [inline Project/MyGameLibrary/Character/CharacterCollection.cs:25]: make these public and static

File: Project/MyGameLibrary/Character/CharacterCollection.cs [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+using System.Text;
+using System.Threading.Tasks;
+using MyGameLibrary.Character;
+using MyGameLibrary.Shop;
+
+namespace MyGameLibrary.Character
+{
+    public class CharacterCollection
+    {
+        public Dictionary<characterID, Character> CharacterDictionary;
```
Comment (HayleyO) [inline Project/MyGameLibrary/Character/CharacterCollection.cs:13]: make this static

File: Project/MyGameLibrary/Character/CharacterCollection.cs [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+        private Dictionary<Items, int> dict4;
+        private Dictionary<Items, int> dict5;
+
+        private Character Wendy;
+        private Character Tony;
+        private Character Ronald;
+        private Character Green;
+        private Character BurgerKing;
+
+        public CharacterCollection() {
```
Comment (HayleyO) [inline Project/MyGameLibrary/Character/CharacterCollection.cs:27]: turn this into a static function and call it on run of the program

File: Project/MyGameLibrary/Character/characterID.cs [hunk 6]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyGameLibrary.Character
+{
+    public enum characterID
```
Comment (HayleyO) [inline Project/MyGameLibrary/Character/characterID.cs:9]: Change the name of this from characterID to CharacterID for naming convention

**Review by HayleyO — CHANGES_REQUESTED** [review]

**Review by hannahfolkertsma — CHANGES_REQUESTED** [review]
Very small things overall to help improve readability 

The only major thing is the constructor. I'm not entirely confident in saying you can't have a static constructor, but it doesn't seem to make sense to me to have one. Overall it might be better to make it an initialize function in general.

**Review by HayleyO — APPROVED** [review]
Thanks so much! Sorry for being a pain great work

**Pull request merged**